src/validation.jl

```diff
@@ -80,13 +81,9 @@ function _validate_ir_instructions_compatibility(
     circuit_instruction_names = map(ix->replace(lowercase(string(typeof(ix.operator))), "_"=>"", "braketsimulator."=>""), circuit.instructions)
     supported_instructions    = Set(map(op->replace(lowercase(op), "_"=>""), supported_operations))
     no_noise = true
-    for name in circuit_instruction_names
-        if name in _NOISE_INSTRUCTIONS
-            no_noise = false
-            if name ∉ supported_instructions
-                throw(ValidationError("Noise instructions are not supported by the state vector simulator (by default). You need to use the density matrix simulator: LocalSimulator(\"braket_dm_v2\").", "TypeError"))
-            end
-        end
+    for name in filter(ix_name->ix_name ∈ _NOISE_INSTRUCTIONS, circuit_instruction_names)
+        no_noise = false
+        name ∉ supported_instructions && throw(ValidationError("Noise instructions are not supported by the state vector simulator (by default). You need to use the density matrix simulator: LocalSimulator(\"braket_dm_v2\").", "TypeError"))
```

Comment:
remove "by default"? can the SV sim ever support noise instructions?

Reply:
This is the original text from the existing v1 sim but yes we can remove